packages/headless/src/modules/cart/dto/addToCart.dto.ts

```diff
@@ -0,0 +1,58 @@
+import { HttpStatus } from "@nestjs/common";
+import { ApiProperty } from "@nestjs/swagger";
+import { IsArray, IsNotEmpty, IsNumber, IsObject, IsOptional, IsString } from "class-validator";
+import { Cart, ResponseItem, CartProduct, addToCartErrorResponse, addToCartRequest, ErrorMessage } from 'models';
+export class AddToCartResponseDto implements Cart {
+    @ApiProperty()
+    @IsString()
+    id?: string;
+
+    @ApiProperty()
+    @IsString()
+    userId: string;
+
+    @ApiProperty({ type: () => [ResItemDto] })
+    @IsArray()
+    items: ResItemDto[];
```

Comment:
ResItemDto i thought this is related to REST. you can change the name to CartItemDto like that